.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           name: Run golangci-lint
           command: |
             export PATH="$GOPATH/bin:$PATH"
-            basecommit=$(git log --first-parent --pretty=format:'%H %D' | egrep '/(master|new-schedulers)' | head -1 | cut -d' ' -f1)
+            basecommit=$(git log --decorate | grep '^commit.*origin/' | sed -n 2p | cut -d' ' -f2)
```

Comment:
I think this will fail if a specific commit is checked out and not a branch... Let me test force pushing.

Reply:
No, works as expected. Still, this feels super hacky, so I wouldn't be surprised if it fails at some point. There doesn't seem to be a 100% reliable solution. :-/

Reply:
If needed, we just use `master`. After all, the goal is for a continuous improvement of the linter state of `master`.

Reply:
In any case, this is good enough for now, we can always change it if we see problems